### PR TITLE
Emil modular file loader

### PR DESCRIPTION
**Pull request by LukasBHolst**

### CODE REVIEW

File: bpm/bpm_extractor.py [hunk 1]
```diff
@@ -1,26 +1,19 @@
 import sys
 import os
 
-# Start of importing the utilities module
-sys.path.insert(0, os.path.abspath("../utilities/"))
-import get_song_id as s_id
+import utilities.get_song_id as s_id
```
Comment (Casperfrc) [inline bpm/bpm_extractor.py]: This import doesn't work and I believe it is not used in the code either.

File: classification/classifier/profile_data_extractor.py [hunk 2]
```diff
@@ -26,32 +28,33 @@ def get_classifier_data(data_file_name):
...
     subprocess.run("rm {}".format(output_file_path), shell=True)
 
-    #list for beautifying code
-    t = [(timbre, timbre_probability), (mood_relaxed, mood_relaxed_probability), 
+    # list for beautifying code
+    t = [(timbre, timbre_probability), (mood_relaxed, mood_relaxed_probability),
          (mood_party, mood_party_probability), (mood_aggressive, mood_aggressive_probability),
          (mood_happy, mood_happy_probability), (mood_sad, mood_sad_probability)
-        ]
+         ]
```
Comment (Casperfrc) [inline classification/classifier/profile_data_extractor.py]: That one space is wrong, imo

File: classification/extractor/t/test_extractor.py [hunk 3]
```diff
@@ -2,12 +2,13 @@
 import sys
 
 from classification.extractor.low_level_data_extractor import make_low_level_data_file
+from filehandler.handle_path import get_absolute_path
 
 def test_song_data_extraction():
+    filename = get_absolute_path("classification/extractor/t/test_extractor/8376-1-1_Demolition_Man_proud_music_preview"
+                                 ".wav")
```
Comment (Casperfrc) [inline classification/extractor/t/test_extractor.py]: I think this line is too long. Why is it two strings?

File: filehandler/handle_audio.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import sys
+import os
+
+
```
Comment (Casperfrc) [inline filehandler/handle_audio.py]: Why is this spacing so big? They are all import statements, right?
I think it should be without spacing like in this:
https://www.python.org/dev/peps/pep-0008/#imports

File: filehandler/handle_path.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import sys
+import os
+
+
+def get_absolute_path(path):
+    dirname = os.path.abspath(os.path.dirname(__file__)) + "/.."
```
Comment (technoten) [inline filehandler/handle_path.py]: What is this wizardry?

File: similarity/t/test_split_song.py [hunk 6]
```diff
@@ -2,12 +2,11 @@
...
+from filehandler.handle_audio import get_MonoLoaded_Song
 
-def test_split_song():
-    dirname = os.path.dirname(__file__)
-    test_filename = os.path.join(
-        dirname, "test_split_song/8376-1-1_Demolition_Man_proud_music_preview.wav")
 
-    segments = split_song(test_filename)
+def test_split_song():
+    song = get_MonoLoaded_Song("similarity/t/test_split_song/8376-1-1_Demolition_Man_proud_music_preview.wav")
```
Comment (Casperfrc) [inline similarity/t/test_split_song.py]: 106 chars in a line, whilst only 79 is permitted according to PEP8.

https://www.python.org/dev/peps/pep-0008/#maximum-line-length

File: video_emotion/facial_recognition/t/test_facial_recognition.py [hunk 7]
```diff
@@ -2,13 +2,12 @@
...
-from video_emotion.facial_recognition.facial_recognition import analyze_video, analyze_frame
 
+from video_emotion.facial_recognition.facial_recognition import analyze_video, analyze_frame
+from filehandler.handle_path import get_absolute_path
 
 def test_facial_recognition_of_at_least_one_face():
-    dirname = os.path.dirname(__file__)
-    test_filename = os.path.join(
-        dirname, "test_facial_recognition/Fun_at_a_Fair.mp4")
+    test_filename = get_absolute_path("video_emotion/facial_recognition/t//test_facial_recognition/Fun_at_a_Fair.mp4")
```
Comment (technoten) [inline video_emotion/facial_recognition/t/test_facial_recognition.py]: Double forward slash here

File: video_emotion/emotionTagger/face_emotion_extraction.py [hunk 8]
```diff
@@ -4,6 +4,7 @@
 import numpy
 from keras.models import load_model
 import glob
+from filehandler.handle_path import get_absolute_path
```
Comment (Casperfrc) [inline video_emotion/emotionTagger/face_emotion_extraction.py]: I think 'simple' imports should be grouped and then more 'advanced' imports with "from whatever import whatever2"

File: video_emotion/emotionTagger/face_emotion_extraction.py [hunk 9]
```diff
@@ -19,10 +20,7 @@ def preprocess_input(x):
 
 def classify_faces(faces):
     # parameters for loading data and images
-    dirname = os.path.abspath(os.path.dirname(__file__))
-    emotion_model_path = os.path.join(
-        dirname,
-        "models/emotion_model.hdf5")
+    emotion_model_path = get_absolute_path("video_emotion/emotionTagger/models/emotion_model.hdf5")
```
Comment (Casperfrc) [inline video_emotion/emotionTagger/face_emotion_extraction.py]: I believe this is more than 79 chars too.

File: video_emotion/emotionTagger/t/test_face_emotion_extraction.py [hunk 10]
```diff
@@ -6,57 +6,44 @@
...
+from filehandler.handle_path import get_absolute_path
+current_directory = "video_emotion/emotionTagger/t/"
 
 
 def test_angry_face():
-    dirname = os.path.abspath(os.path.dirname(__file__))
-    filename = os.path.join(
-        dirname,
-        "test_face_emotion_extraction_data/angry1.png")
+    filename = get_absolute_path(current_directory + "test_face_emotion_extraction_data/angry1.png")
```
Comment (Casperfrc) [inline video_emotion/emotionTagger/t/test_face_emotion_extraction.py]: More than 79 chars

File: video_emotion/emotionTagger/t/test_face_emotion_extraction.py [hunk 11]
```diff
@@ -6,57 +6,44 @@
 from video_emotion.emotionTagger.face_emotion_extraction import classify_faces
 from video_emotion.emotionTagger.face_emotion_extraction import get_labels
 from video_emotion.emotionTagger.face_emotion_extraction import preprocess_input
```
Comment (Casperfrc) [inline video_emotion/emotionTagger/t/test_face_emotion_extraction.py]: I think according to PEP8 we should find a way to write our imports in less than 79 chars.

File: video_emotion/emotionTagger/t/test_face_emotion_extraction.py [hunk 12]
```diff
@@ -84,19 +71,14 @@ def test_preprocess_input():  # expermentially we have reached that it should ou
...
-    fileName3 = os.path.join(
-        dirname,
-        "test_face_emotion_extraction_data/happy3.png")
-    img1 = cv2.imread(fileName1)
-    img2 = cv2.imread(fileName2)
-    img3 = cv2.imread(fileName3)
+
+    filename1 = get_absolute_path(current_directory + "test_face_emotion_extraction_data/happy1.png")
+    filename2 = get_absolute_path(current_directory + "test_face_emotion_extraction_data/happy2.png")
+    filename3 = get_absolute_path(current_directory + "test_face_emotion_extraction_data/happy3.png")
```
Comment (Casperfrc) [inline video_emotion/emotionTagger/t/test_face_emotion_extraction.py]: 79 Chars

File: video_emotion/facial_recognition/facial_recognition.py [hunk 13]
```diff
@@ -3,17 +3,14 @@
 import numpy as np
 import cv2
 
+from filehandler.handle_path import get_absolute_path
```
Comment (Casperfrc) [inline video_emotion/facial_recognition/facial_recognition.py]: Don't think there should be spacing between this.

File: video_emotion/facial_recognition/t/test_facial_recognition.py [hunk 14]
```diff
@@ -2,13 +2,12 @@
 
 import cv2
 
-from video_emotion.facial_recognition.facial_recognition import analyze_video, analyze_frame
 
+from video_emotion.facial_recognition.facial_recognition import analyze_video, analyze_frame
```
Comment (Casperfrc) [inline video_emotion/facial_recognition/t/test_facial_recognition.py]: Spacing is weird here too. Should be grouped.

File: video_emotion/facial_recognition/t/test_facial_recognition.py (hunk 7 above)
Comment (Casperfrc) [inline video_emotion/facial_recognition/t/test_facial_recognition.py]: 79 chars

File: classification/extractor/t/test_extractor.py (hunk 3 above)
Comment (Areedtz) [reply classification/extractor/t/test_extractor.py]: :+1: to Casper's comment.

File: similarity/t/test_split_song.py (hunk 6 above)
Comment (Areedtz) [reply similarity/t/test_split_song.py]: :+1: to Casper's comment.

File: video_emotion/emotionTagger/t/test_face_emotion_extraction.py (hunk 11 above)
Comment (Areedtz) [reply video_emotion/emotionTagger/t/test_face_emotion_extraction.py]: :+1: to Casper's comment

File: video_emotion/emotionTagger/t/test_face_emotion_extraction.py (hunk 10 above)
Comment (Areedtz) [reply video_emotion/emotionTagger/t/test_face_emotion_extraction.py]: :+1: to Casper's comment

**Pull request merged**